client/web/src/theme.test.ts

```diff
@@ -0,0 +1,131 @@
+// causes false positive on act()
+/* eslint-disable @typescript-eslint/no-floating-promises */
+
+import { renderHook, act } from '@testing-library/react-hooks'
+
+import { ThemePreference, useTheme } from './theme'
+
+// Don't test reacting to system-wide theme changes, for simplicity. This means that
+// observeSystemIsLightTheme's initial value will be used, but it will not monitor for subsequent
+// changes.
+jest.mock('../../shared/src/util/useObservable', () => ({
```

Comment:
Is it possible to use the absolute path here for cross-package dependency similar to [what we do](https://github.com/sourcegraph/sourcegraph/pull/24227/files#diff-13cc7ed21d89a6f4b5ff73b640475d3da81d8a649610a6f63d99fdbba0ae3047R4) in source files?

```suggestion
jest.mock('@sourcegraph/shared/src/util/useObservable', () => ({
```